### PR TITLE
issue #685 solved image viewer added in gallery section

### PR DESCRIPTION
**Pull request by theonlyjunaid**
Issue #685 

my solution preview

https://user-images.githubusercontent.com/95287515/217584409-e4e0c19e-226f-40bf-a834-bb6b9496a74b.mp4

### CODE REVIEW

File: src/components/media/PhotoGalleryModal.tsx [hunk 1]
```diff
@@ -37,6 +42,45 @@ const PhotoGalleryModal = ({ setShowPhotoGalleryModal }: PhotoGalleryModalProps)
...
+        ShowImage ? ( 
+          <div className="fixed inset-0 z-50 flex items-center justify-center bg-black bg-opacity-75">
+            <div className="relative w-full h-full">
+              <div className="absolute top-2 right-2 z-50">
+                <button
+                  onClick={() => setShowImage('')}
+                  className="bg-white hover:bg-gray-500 bg-opacity-50 rounded-full  duration-300 ease-in-out p-2"
+                >
+                  <svg
+
```
Comment (vnugent) [inline src/components/media/PhotoGalleryModal.tsx:48]: can you please use an icon for example `x-mark` from Heroicons https://heroicons.com? The library is already in package.json.

Comment (vnugent) [reply src/components/media/PhotoGalleryModal.tsx:48]: also can you run `yarn lint` and fix linting errors?  then we're good to merge!

**Review by vnugent — CHANGES_REQUESTED** [review]

File: src/components/media/PhotoGalleryModal.tsx (hunk 1 above)
Comment (theonlyjunaid) [reply src/components/media/PhotoGalleryModal.tsx:48]: Sure I will fix that

Comment (vnugent) [reply src/components/media/PhotoGalleryModal.tsx:48]: Example:

```javascript
import { XMarkIcon } from '@heroicons/react/24/outline'

...
<XMarkIcon className='w-6 h-6' />
```

**Comment by vnugent** [comment]
@all-contributors add @theonlyjunaid for code

**Pull request merged**